src/insights_client/metrics.py

```diff
@@ -9,10 +9,10 @@
 
 
 def _proxy_settings(rhsm_config):
-    hostname = rhsm_config.get("server", "proxy_hostname").strip()
-    port = rhsm_config.get("server", "proxy_port").strip()
-    user = rhsm_config.get("server", "proxy_user").strip()
-    password = rhsm_config.get("server", "proxy_password").strip()
+    hostname = rhsm_config.get("server", "proxy_hostname", fallback=None)
+    port = rhsm_config.get("server", "proxy_port", fallback="")
+    user = rhsm_config.get("server", "proxy_user", fallback="")
+    password = rhsm_config.get("server", "proxy_password", fallback="")
```

Comment:
Do we really want to get rid of the _strip_ calls? They should remove any trailing whitespace that may occur there by accident.

It wouldn’t work for _None_ fallback of hostname, but that can have an empty string fallback too. Empty strings are falsey in conditionals.